ui/profiles/sheets/OtherBacklogSheet.tsx

```diff
@@ -44,7 +45,7 @@ export const OtherBacklogSheet = ({
       snapPoints={['50%', '90%']}
       index={-1}
       animatedIndex={otherProfileBackdropAnimatedIndex}
-      backgroundStyle={{ backgroundColor: c.olive, borderRadius: s.$4, paddingTop: 0 }}
+      backgroundStyle={{ backgroundColor: c.olive, borderRadius: 50, paddingTop: 0 }}
```

Comment:
We should use the spacing variables (`s.$1`, `s.$5`, etc etc) for things like this, just to be consistent with the rest of the codebase